LibreHardwareMonitorLib/Interop/D3dkmdt.cs

```diff
@@ -0,0 +1,48 @@
+﻿// This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0.
+// If a copy of the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
+// Copyright (C) LibreHardwareMonitor and Contributors.
+// All Rights Reserved.
+
+// ReSharper disable IdentifierTypo
+// ReSharper disable InconsistentNaming
+
+using System.Runtime.InteropServices;
+
+namespace LibreHardwareMonitor.Interop
+{
+    internal static class D3dkmdt
+    {
+        internal enum DXGK_ENGINE_TYPE
```

Comment:
Can we just keep the comments for the engine types? I think those were helpful.

Maybe add as a hover tooltip for the sensor?